src/main/java/seedu/club/logic/commands/Command.java

```diff
@@ -40,4 +40,26 @@ public static String getMessageForMemberListShownSummary(int displaySize) {
     public void setData(Model model, CommandHistory history, UndoRedoStack undoRedoStack) {
         this.model = model;
     }
+
+    /**
+     * Requires user to login before proceeding
+     */
+    protected Boolean requireToLogIn () {
+        if (model.getLoggedInMember() == null) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    /**
+     * Requires user to Sign Up
+     */
+    protected Boolean requireToSignUp () {
```

Comment:
Can throw Command_Exception here instead of return boolean. See requireNonNull function, it should be the same format